automation_testing/pages/DataViewPage.ts

```diff
@@ -143,4 +140,46 @@ export class DataviewPage {
 		}
 		return createdDates;
 	}
+
+	async validateContainsKeyword(titles: string[], keyword: string): Promise<void> {
+		for (const title of titles) {
+			expect(title.toLowerCase()).toContain(keyword.toLowerCase());
+		}
+	}
+
+	async validateContainsMoreThanOneKeyword(titles: string[], keywords: string[]): Promise<void> {
+
+
+		for (const title of titles) {
+			let isContaining = false;
```

Comment:
I don't think having the flag here fixes the issue. Let's assume titles has a length of 3, and keywords has a lenght of 2: titles[0] includes keywords[0] then the flag is changed to true, but then even if titles[0] doesn't include keywords[1] it will still be true because the flag resets to false until it finishes with all the keywords on title[0]. Let me know if we need to have a meeting to explain it a bit better.